docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -0,0 +1,163 @@
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-ID='<YOUR_CLIENT_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-SECRET='<YOUR_CLIENT_SECRET>' \
+    -e ZEEBE_CLIENT_CLOUD_REGION='<YOUR_CLUSTER_REGION>' \
+    -e CAMUNDA_OPERATE_CLIENT_URL='https://<region>.operate.camunda.io/<cluster-id>' \
+    -e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local' \
+        camunda/connectors-bundle:<desired-version>
+```
+
+### Option 2: Build your own runtime
+
+Use this option if you need to make code modifications.
+
+1. Ensure `docker` is installed.
+2. Clone [https://github.com/camunda/connectors](https://github.com/camunda/connectors).
+3. Go to `<connectors-root>/bundle/default-bundle`.
+4. Build a Connector image, e.g. `docker build -f Dockerfile -t camunda/connectors-bundle:<desired-version> .`.
+5. Run the same `docker run ...` command as in [Option 1](#option-a-get-connectors-runtime-from-docker-registry).
+
+### Explanation
+
+Note the line `-e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local'`. This line means we have to override
+`CONNECTOR_X_TYPE` with a given type. In this case, we want to register a local Self-Managed HTTP REST Connector as `io.camunda:http-json:local`.
+
+The `X` is normalized to the environment variable Connector name. For example, the [HTTP REST Connector](https://github.com/camunda/connectors/blob/main/connectors/http/rest/src/main/java/io/camunda/connector/http/rest/HttpJsonFunction.java#L33)
+`HTTP REST` name becomes `HTTP_REST`, or the [Kafka Consumer Connector](https://github.com/camunda/connectors/blob/main/connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java#L20) name
+becomes `KAFKA_CONSUMER`. Therefore, to override it one would need to pass in the `CONNECTOR_KAFKA_CONSUMER_TYPE=xxx` environment variable.
+
+## Preparing element template for hybrid mode
+
+As mentioned, to relate Connector element templates with Connectors runtime, you must modify the task definition type.
+
+To do this, take the following steps:
+
+1. Obtain a copy of the element template you wish to override. All latest versions of the official element
+   templates can be found in the [official Connectors repository](https://github.com/camunda/connectors) at path `connectors/<desired connector>/element-templates/`.
+2. Modify the `value` to the desired new type of the property. Use `zeebe:taskDefinition:type` for outbound Connectors, or `inbound.type` for inbound ones.
+3. Publish the new element template, and use it in your BPMN diagram.
+
+There are several options to deliver element templates to the target user:
+
+### Option 1: Hide task definition type value
+
+Use this option when you plan to clearly indicate that a specific Connector will only be used in a specific use-case.
+Otherwise, users might be confused between two of the same Connector types.
+
+For example, if you defined `CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local'` in runtime, you must implement the following in the
```

Comment:
In Connectors runtime, specifically here? May need to add this to my Connector(s) runtime issue.

Reply:
reworded a bit